docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -591,7 +602,8 @@ def delete_project_file_version_permanently(
     return versions_to_delete
 
 
-def get_stored_package_ids(project_id: str) -> set[str]:
+def get_stored_package_ids(project: qfieldcloud.core.models.Project) -> set[str]:
+    project_id = project.id
```

Comment:
I use the `project_id = str(project.id)` to minimize the diff.

These funcions will be gone in 3 months or so anyways.

Reply:
Is there any added value to add a warning about this deprecation ?

Reply:
There is, check the #1065 PR.

Reply:
The current PR is not proposing the deprecation yet. But in the PR that actually introduces the deprecation I have added a deprecation warning.

https://github.com/opengisch/QFieldCloud/blob/QF-2760-files/docker-app/qfieldcloud/core/utils2/storage.py#L29-L47